app/src/main/java/pl/netigen/drumloops/rock/features/listmusic/data/local/dao/AudioDao.kt

```diff
@@ -0,0 +1,19 @@
+package pl.netigen.drumloops.rock.features.listmusic.data.local.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import pl.netigen.drumloops.rock.features.listmusic.data.local.model.AudioCached
+
+@Dao
+interface AudioDao {
+    @Query("SELECT * FROM AudioCached")
+    suspend fun getAudio(): List<AudioCached>
+
+    @Query("SELECT * FROM AudioCached WHERE isLike = 1")
+    suspend fun getLikeAudio(): List<AudioCached>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
We want to save but OnConflictStrategy.IGNORE, maybe it should named saveIfNotexist, or changed to  OnConflictStrategy.REPLACE

Reply:
this function adds new from api to update will be different